templates/index.html

```diff
@@ -59,7 +59,7 @@
 
       <div class="sri-test" id="sri-badge">
         <link rel="stylesheet" href="/badge/will-pass.css">
-        <link rel="stylesheet" href="/badge/must-fail.css" integrity="sha256-INVALIDHASH">
+        <link rel="stylesheet" href="/badge/must-fail.css" integrity="type:invalid/type sha256-INVALIDHASH">
```

Comment:
I see what you mean. Yes, we should only have one failure condition here. No need to include the content-type as well.